### PR TITLE
Fix tiny typo on resource limit description for all available languages

### PR DESCRIPTION
**Pull request by kessiler**

### CODE REVIEW

**Comment by cheld** [comment]
The typo was also translated - cool :-) 

The englisch file is generated. Please fix also 
https://github.com/kubernetes/dashboard/blob/dbbcb06995321e75f7e67cac83fb8a5f044d1374/src/app/frontend/resourcelimit/detail/detail.html#L37

Thanks for the PR

**Comment by floreks** [comment]
You have to fix HTML file and run `gulp build` it will update our translation files. You should not directly change them only translated strings can be manually updated. Part `xxx` in the `<translation>xxx</translation>` tag.

**Comment by kessiler** [comment]
done @cheld and @floreks

File: src/app/frontend/resourcelimit/detail/detail.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     </kd-resource-card-header-column>
     <kd-resource-card-header-column size="small"
                                     grow="1">
-      [[Min|]Resource Limits min entry.]]
+      [[Min]Resource Limits min entry.]]
```
Comment (maciaszczykm) [inline src/app/frontend/resourcelimit/detail/detail.html]: It should be `[[Min|Resource Limits min entry.]]`

Comment (kessiler) [inline src/app/frontend/resourcelimit/detail/detail.html]: done @maciaszczykm

**Pull request merged**